src/app/map/page.tsx

```diff
@@ -1,24 +1,16 @@
-import Link from "next/link";
 import { type Metadata } from "next";
-import Button from "~/clients/components/Button";
-import Text from "~/clients/components/Text";
+import { GoogleMaps } from "~/clients/components";
 
 export const metadata: Metadata = {
   title: "Map",
   description: "Generated by create-t3-app",
 };
 
 const MapPage = () => {
-  return (
-    <>
-      <Text fz="lg" fw={700} mb={20}>
-        Map Page
-      </Text>
-      <Link href={`/`}>
-        <Button variant="light">Back Home</Button>
-      </Link>
-    </>
-  );
+  // [TODO] get current latitude and longitude from geolocation
+  const address = "601 W Cordova St, Vancouver, BC V6B 1G1";
+
+  return <GoogleMaps address={address} />;
 };
```

Comment:
@devjoylee Imo, we don't have to pass `address` as props and instead we can declare `const address` inside `GoogleMaps` component.

Reply:
We still need a code for current geolocation, probably as a util function or just in this map page?
I made the `address` prop as temporary variable instead of geolocation code just for now (only to present the pin). 
So yes! it's true that we don't need to pass as props. will modify them later with geolocation.

I didn't put it inside `components/GoogleMaps` because I wanted to make the geolocation as a separated function,
For example `getCurrentLocation` ??  or we can just include in the component. Let's think about it.